src/Hadolint/Rules.hs

```diff
@@ -506,24 +520,18 @@ npmVersionPinned = instructionRule code severity message check
         \<package>@<version>`"
     check (Run (Arguments args)) = all versionFixed (packages args)
     check _ = True
-    packages :: [String] -> [String]
```

Comment:
out of curiosity.. why do you remove type signatures?

Reply:
They were very noisy in this context, they were making the reading of the code quite difficult for me. I can bring them back if you find the useful.

Reply:
OK, no problem